garnet.py

```diff
@@ -468,6 +471,9 @@ def main():
     parser.add_argument("--pipeline-pnr", action="store_true")
     parser.add_argument("--generate-bitstream-only", action="store_true")
     parser.add_argument('--pe', type=str, default="")
+    parser.add_argument('--mem-ratio', type=int, default=4)
+    parser.add_argument('--num-tracks', type=int, default=5)
+    parser.add_argument('--tile-layout-option', type=int, default=0)
```

Comment:
Use argparse flags

Reply:
Do you mean adding a shorter flag like `parser.add_argument('-m', '--mem-ratio', type=int, default=4)` ?

Reply:
No, I mean using action `store_true`, see `--generate-bitstream-only`. This is only valid for boolean flags.